src/main/java/com/ldtteam/domumornamentum/block/ModBlocks.java

```diff
@@ -87,6 +89,9 @@ public ShingleBlock getShingle()
         return new ArrayList<>(ModBlocks.TIMBER_FRAMES);
     }
 
+    @Override
+    public PillarBlock getPillar() {return ModBlocks.PILLAR;}
```

Comment:
Formatting. See the other methods in this class.

Reply:
huh. my IDE must have automatically moved these all into one liners. Thought I was following the format there.